pkg/Library/App/Worker/Arcs.js

```diff
@@ -89,6 +89,8 @@ const handleServiceCall = async msg => {
   socket.sendVibration({kind: 'serviceResult', sid: msg.sid, data});
 };
 
+// public API
```

Comment:
This is great! ...`init` should go here too please

Reply:
done